### PR TITLE
Announcement for new system and cloudstack retirement

### PR DESCRIPTION
**Pull request by harsha5500**
Hej,

I have added a short writeup for the retirement of cloud stack. Please feel free to reduce or modify (or even delete) the proposed announcement. It is not too technical at this point as I am still figuring out rancher but please add anything you deem necessary. 

Thank you.

Harsha

### CODE REVIEW

**Review by saffronjam — COMMENTED** [review]
Cloud stack & cloudstack --> CloudStack 

Otherwise it looks good! 😄

File: hugo/content/News/2024-04-29.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+While the purpose for a while, the number of users in the cloud increased and the demand for the least avaliable source, the GPU also increased. We needed a better way to distribute them.
+
+While cloudstack is good for a very large planned cluster, it required significant amount of effort to maintain. Cloud stack has limited community and is still lacks some of the feature for a cluster like ours, which is based on donated hardware (We don't always now what we get :smiley face: ) . It uses an extensive combination of virtual networks and its own virtual machines in order to orchestrate various clusters and VMs in the cluster. Things really came to head when we had the incident in March which required a immediate shutdown following a failed upgrade. At this time the entire cluster became unstable.
+
+## Hello KubeVirt
+
+Fortunately, Emil was testing a new method to create VMs within a kubernetes cluster. This allowed us to run the entire infrastructure as pure kuberentes system. It also has the added advantage of running different loads as scheduled nodes with or without GPUs. This allows for quicker rotation of GPUs and possibly fairer sharing.
```
Comment (pierrelefevre) [inline hugo/content/News/2024-04-29.md:23]: Kubernetes*

**Comment by harsha5500** [comment]
Got it will change the spelling and test on Hugo

**Review by pierrelefevre — CHANGES_REQUESTED** [review]

**Comment by harsha5500** [comment]
Fixed the changes and pushed to the harsha branch.

File: hugo/content/News/2024-04-29.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: My Overwritten Name
```
Comment (pierrelefevre) [inline hugo/content/News/2024-04-29.md:2]: Should be your actual title

File: hugo/content/News/2024-04-29.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+weight: 2 # Order in the list
```
Comment (pierrelefevre) [inline hugo/content/News/2024-04-29.md:3]: We don't use weight in the news tab since they are sorted by filename (ISO date)

**Review by pierrelefevre — CHANGES_REQUESTED** [review]

**Comment by pierrelefevre** [comment]
Sorry missed the top metadata stuff before but otherwise LGTM, you can just check how it looks in the other blog posts like https://raw.githubusercontent.com/kthcloud/docs/main/hugo/content/News/2024-03-22.md

**Comment by harsha5500** [comment]
Oops! Fixed the title and pushed again

**Review by pierrelefevre — APPROVED** [review]
LGTM +1

**Comment by pierrelefevre** [comment]
Hmm I thought this would have been merged already!

**Pull request merged**